flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/AbstractChangelogState.java

```diff
@@ -34,10 +34,13 @@
 abstract class AbstractChangelogState<K, N, V, S extends InternalKvState<K, N, V>>
         implements InternalKvState<K, N, V> {
     protected final S delegatedState;
+    protected N currentNamespace;
```

Comment:
`currentNamespace`  is not initialized, so it is possible to be null? 

From the context, it seems namespace can not be null, so it is better to check its non-null when using it?
`checkNotNull(namespace, "Namespace");` in `AbstractKeyedStateBackend#getPartitionedState`

Reply:
Yes, you are right.